docs/setup/installation.md

```diff
@@ -2,50 +2,87 @@
 title: Installation
 ---
 
-SourceCred currently support running from a Node environment or using Docker.
-_In a future release we're looking to add easier options._
+The easiest way to install SourceCred is to fork the [example repo](https://github.com/sourcecred/example-instance). This repo contains scripts that will install SourceCred locally and set up an example instance of SourceCred. This instance can then be configured to include just the plugins you want and point to the data sources you care about.
 
-## Node environment
+If you want to build SourceCred from source (for instance to get bleeding edge features) or to develop on it, see the [Development Setup instructions](#development-setup).
 
-The Node environment is useful for development and running from source code.
+You can also run SourceCred from a [Docker image](#using-docker)  (useful for automated setups and canonical instances). 
 
-Make sure you've installed:
-- Node version 10 or 12.
-- Yarn version 1.7 or greater.
+## Installing from example instance
 
-Releases can be downloaded from https://github.com/sourcecred/sourcecred/releases.
-Or you can download the latest `master` branch if you're feeling brave.
+1. Clone the `example-instance` repo from SourceCred’s GitHub:
 
-Then run the following commands to download dependencies and compile the CLI
-(backend).
+`git clone https://github.com/sourcecred/example-instance.git`
 
-```bash
-cd /path/to/extracted/code
-yarn install
-yarn backend
-```
+2. Install SourceCred and its dependencies
 
-From here you can use the CLI.
+If you don't have it, install [yarn](https://classic.yarnpkg.com/lang/en/). Then run yarn to install SourceCred and its dependencies.
 
-```bash
-node bin/sourcecred.js help
+`yarn` 
+
+That's it! The installed instance with come with the latest stable release and default configurations, including all plugins.
```

Comment:
This isn't really enough--presumably the user isn't interested in having a SC instance that only shows them example data. They need more help to get it set up for their own data sources. This is basically per-plugin though: maybe we should link to a setup page for each plugin?

Reply:
Had the same thought. Was thinking of putting instructions for selecting and configuring plugins here. But then thought that would conflict with what I assumed was the decided upon structure:

![image](https://user-images.githubusercontent.com/1634777/87816737-ab7da880-c81c-11ea-97b6-9fa45ba152e2.png)

Seems like setting up plugins would logically fall under configuration. For configuration, was imagining a high-level page, similar to installation, with links to separate pages for each plugin (there's a lot for some plugins, and don't want to make the page too long). Was thinking then we could have a separate "Getting Started Guide" page that walks you through everything, including setting up a simple plugin config (just Discourse, as it requires no permissions stuff?). 

However, if we're making the installation page this short, it might make sense to fold instructions into a more 'getting started guide' style doc, with general high level instructions on how to configure plugins? 

Related (just dumping this here but can break out into separate convo if desired): do we have two separate pages for each plugin, one on configuring it and another describing how cred flows? Torn on this. Perhaps actually just collecting all the plugin config in the Configuration page makes most sense actually? Then there's not confusion around two pages called "Discord" (or similar).